src/ios/UsabillaCordova.swift

```diff
@@ -24,29 +23,25 @@ import Usabilla
                     self.masks = value as? [String]
                 } else if (key == "MASK_CHAR") {
                     self.maskChar = value as? String
-                } else {
-                    if (value is String) {
-                        arguments[key] = value as? String
-                    } else if (value is Bool) {
-                        arguments[key] = value as? Bool
-                    }                    
+                } else if (key == "CUSTOM_VARS") {
+                    self.customVariables = value as? [String : Any]
```

Comment:
we are missing a generic `else` statement
Is it worth tracking/logging if they key is unexpected?